images/autocertbot/autocert.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+"""
+Helper script to run certbot inside a kubernetes cluster.
+
+certbot expects contents of /etc/letsencrypt to be persistent
+across runs, so it knows when to renew certificates & when to leave
+them alone. This is a problem in Kubernetes, since we try to avoid
+having persistent disks unless we must.
+
+This script saves / restores the contents of /etc/letsencrypt into
+a Kubernetes secret object, thus letting certbot operate unchanged
+without needing any persistent storage.
+
+This script runs as a sidecar to an nginx container that has webroot
```

Comment:
I think this comment is outdated now that we use traefik. Can we replace it with the equivalent for the traefik setup or remove it?

Reply:
I agree! I just wanted to push this up before I left. I think this script should be renamed secret-sync or something like that, and redocumented. I'll do that tomorrow.

Reply:
Redocumented!